packages/harpguru-core/src/components/harp-cell/harp-cell.tsx

```diff
@@ -64,15 +49,10 @@ export const HarpCell = ({ yxCoord }: HarpCellProps): React.ReactElement => {
   }
 
   return (
-    <LongPressGestureHandler
```

Comment:
Raise issue to reintroduce long hold feature.

Add CHANGELOG entry for it's removal.

Reply:
Done & raised at https://github.com/js-jslog/harpguru/issues/72